scripts/cycles.js

```diff
@@ -17,3 +17,27 @@ function chooseWeekDays() {
     }
   }
 }
+
+let n = 1000;
```

Comment:
Why is this variable out of scope of the function it used in?

Reply:
This is my mistake. I just wrote down my thoughts in order.